core/templates/dev/head/editor/EditorServices.js

```diff
@@ -59,7 +59,17 @@ oppia.factory('explorationData', [
             $log.info('Retrieved exploration data.');
             $log.info(response.data);
 
-            explorationData.data = response.data;
+            // FIXME: Update this value based on what exactly is being sent from
+            // the backend (corresponding to the ‘current exploration’ data).
+            if (response.data.currentExplorationData) {
```

Comment:
This seems strange. Why would you want either the whole response.data or just one key from it?